tests/test_gin.py

```diff
@@ -28,7 +29,7 @@
     HAVE_PARAMETERIZED = False
```

Comment:
I think `parameterized` should be listed in `requirements-dev.txt`, so we should be able to expect that it is downloaded and shouldn't need this import catch

Reply:
It is [https://github.com/catalystneuro/nwb-conversion-tools/blob/add_phy_gin/requirements-dev.txt#L19](https://github.com/catalystneuro/nwb-conversion-tools/blob/add_phy_gin/requirements-dev.txt#L19) but it's still needed when running pytest locally (not on CI) in case the developer has not installed it